common/common/src/main/java/io/helidon/common/FeatureCatalog.java

```diff
@@ -116,6 +116,10 @@
                     .nativeSupported(false)
                     .flavor(HelidonFlavor.SE)
         );
+        addSe("io.helidon.metrics.micrometer",
```

Comment:
Did you check native image support?

Reply:
It worked for me in SE which is all the PR has so far. Did you see something that worried you about native image support?